content/en/docs/concepts/storage/volumes.md

```diff
@@ -1118,6 +1118,17 @@ spec:
 
 More examples can be found [here](https://github.com/kubernetes/examples/tree/master/staging/volumes/vsphere).
 
+#### CSI Migration
+
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+
+The CSI Migration feature for vsphereVolume, when enabled, shims all plugin operations
+from the existing in-tree plugin to the `csi.vsphere.vmware.com` Container
+Storage Interface (CSI) Driver. In order to use this feature, the [vSphere CSI
```

Comment:
I'd consider using `{{< glossary_tooltip text="Container Storage Interface" term_id="csi" >}}` to get a glossary tooltip and hyperlink.